### DISCUSSION TITLE
sir g jb df_linear karty hai tb  x=0 khu ayti hy

### DISCUSSION THREAD

**Discussion by waqarahmad45** [AammarTufail/python_ka_chilla2023#77] [Q&A]
![image](https://user-images.githubusercontent.com/121750424/213443584-4a222cdd-5604-417b-bf31-fb9a95a2f783.png)

**Comment by waqarahmad45** [comment]
bhai is mai x =0 ayta hy

**Reply by Muhammad-Ali-Butt** [reply]
@waqarahmad45 bhai is say upar walay code ka screenshot send krna

**Reply by bakhtawar-ishaq** [reply]
oper jahan dataset load kiya ha uske neche ye code run karein
df_linear=df[["age","fare"]]

ab dataset update ho gaya ha.

**Comment by waqarahmad45** [comment]
Thanks bhai
